backend/app/Http/Controllers/AuthController.php

```diff
@@ -58,7 +59,13 @@ public function login(Request $request)
             'user' => $user,
             'token' => $token
         ];
-
         return response($response, 201);
     }
+
+    public function logout(Request $request)
```

Comment:
```suggestion
    public function logout()
```
Remove `Request $request` because we are not using it